lms/djangoapps/student_account/views.py

```diff
@@ -421,6 +425,13 @@ def account_settings_context(request):
     user = request.user
 
     year_of_birth_options = [(unicode(year), unicode(year)) for year in UserProfile.VALID_YEARS]
+    try:
+        user_orders = get_user_orders(user)
+    except Exception as err:  # pylint: disable=broad-except
+        log.warning('Error fetching order history from otto: %s', err.message)
```

Comment:
If you use `log.exception` the stacktrace will be logged too, which would be helpful when debugging problems here.